dist/index.html

```diff
@@ -51,6 +51,8 @@
       </div>
     </div>
   </nav>
+  <main id="categories"></main>
+
   <footer>Exampliary contenct that we going to update later according to our theme</footer>
```

Comment:
I think it would be better to add some styles to the footer for getting a better design for users.
![7](https://user-images.githubusercontent.com/85481456/232785268-b3cff54b-89d0-482b-aaaf-114dbe34cc28.png)